core/templates/dev/head/services/PlaythroughServiceSpec.js

```diff
@@ -249,16 +249,23 @@ describe('Playthrough service', function() {
       });
   });
 
-  describe('Test whitelisting functions', function() {
-    it('should not record learner actions for blacklisted exps', function() {
-      this.PlaythroughIssuesService.initSession(
-        this.blacklistedExpId, this.expVersion, [this.whitelistedExpId]);
+  describe('Test recording disabled', function() {
```

Comment:
maybe:
```suggestion
  describe('Test playthrough recording disabled', function() {
```

Actually.. can this suite be merged with the above `describe` itself? I think it makes sense to do that.

Reply:
Hmm, I think I'll just refactor this whole spec in a separate PR since there's a lot I'd like to see updated. Does that sound OK to you?

Reply:
Nevermind, I ended up doing the refactoring in this PR :) PTAL!